client/src/styles/legacy/_style.scss

```diff
@@ -1320,6 +1320,25 @@ html, body {
   }
 }
 
+// Tree batch action parameter fields
+#BatchActionParameters {
```

Comment:
Can we target this without using an ID selector?

Reply:
Yeah I could, but to keep the css from impacting other areas, I think I would need to add at least one class to the `#BatchActionParameters` LiteralField.

For that I could eight create a class called `cms-batch-action-parameters` (to match with the existing `cms-batch-actions` naming) and keep the current nesting of the SCSS rules. Or, if we wanted, I could call the class `action-parameters`, and we could nest all of the rules under `cms-batch-actions` (so the resulting generated rule would be `.cms-batch-actions .action-parameters .form__field-description`). Which would keep everything together in the SCSS file, but also add an extra level of specificity (which is unneeded)

What do you think?

Reply:
> I think I would need to add a class to the #BatchActionParameters LiteralField (in the CMS PR)

That sounds like a good solution to me. Looking at the other styles in that file, it seems that nesting `.action-parameters` inside `.cms-batch-actions` would be most consistent (assuming the markdown is in fact nested that way).